### ISSUE TITLE
Forward multiple ports from docker container

### ISSUE THREAD

**Issue by jas99** [containous/traefik#414]
How can I forward multiple ports from docker container (i.e more then one to diff frontends)?

Thx

**Comment by jas99** [comment]
Hi let me rephrase; 

myapp container exposing port 5000,8000 and I wanna redirect like this with traefik
5000 --------------------> myapp.localhost:5000
8000 --------------------> myapp.localhost:7000
 
thxs
@emilevauge

**Comment by emilevauge** [comment]
Duplicate #61

**Comment by jas99** [comment]
thx for pointing that out
